src/providers/gdal/qgsgdaldataitems.h

```diff
@@ -37,4 +39,15 @@ class QgsGdalLayerItem : public QgsLayerItem
     QString layerName() const override;
 };
 
+//! Provider for GDAL root data item
+class QgsGdalDataItemProvider : public QgsDataItemProvider
+{
+  public:
+    QString name() override { return QStringLiteral( "GDAL" ); }
+
+    int capabilities() override { return QgsDataProvider::File | QgsDataProvider::Dir; }
```

Comment:
Did you omit QgsDataProvider::Net intentionally? (I think it was there in the previous implementation)

Reply:
Fixed.